frontend/app/component/form/polling_station_choice/PollingStationChoiceForm.test.tsx

```diff
@@ -1,7 +1,8 @@
 import { describe, expect, test } from "vitest";
 import { userEvent } from "@testing-library/user-event";
-import { render, screen } from "app/test/unit";
+import { render, screen, within } from "app/test/unit";
 import { PollingStationChoiceForm } from "app/component/form/polling_station_choice/PollingStationChoiceForm.tsx";
+import { PollingStationProvider } from "@kiesraad/api";
 
 describe("Test PollingStationChoiceForm", () => {
   test("Form field entry and buttons", async () => {
```

Comment:
Something is going on with the `expect` on `toBeInvalid()` in this test.

If I execute these steps in a browser myself, `aria-invalid="false"` never changes to `aria-invalid="true"`.

If I change the `expect` to `expect(pollingStation).not.toBeInvalid();`, the test fails with the following output:
```
Error: expect(element).not.toBeInvalid()

Received element is currently invalid:
  <input
  aria-invalid="false"
  id="pollingStation"
  maxlength="6"
  name="number"
  pattern="\d+"
  type="text"
  value="abc"
/>
```

So the element is invalid, even though `aria-invalid="false"`?

Reply:
Now that the pattern is removed, I think this is expected behavior. If not, we should open an issue